x-pack/plugin/watcher/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/watcher/activate_watch/10_basic.yml

```diff
@@ -49,16 +49,6 @@
 
   - match: { status.state.active : false }
 
-  - do:
```

Comment:
These test already made the corresponding get watch api calls,
so only need to remove the search api calls here.